schemas/targets.schema.json

```diff
@@ -0,0 +1,274 @@
+                      "enum": ["rsa"]
+                    },
+                    "keyval": {
+                      "type": "object",
+                      "required": ["public"],
+                      "additionalProperties": false,
+                      "properties": {
+                        "public": {
+                          "type": "string",
+                          "pattern": "^-----BEGIN PUBLIC KEY-----"
+                        }
+                      }
+                    },
+                    "scheme": {
+                      "enum": ["rsassa-pss-sha256"]
+                    }
+                  },
+                  "required": [
+                    "keyid_hash_algorithms",
+                    "keytype",
+                    "keyval",
+                    "scheme"
+                  ]
+                }
+              }
+            },
+            "roles": {
+              "type": "array",
+              "items": [
+                {
+                  "type": "object",
+                  "additionalProperties": false,
+                  "properties": {
+                    "keyids": {
+                      "type": "array",
+                      "items": [
+                        {
+                          "type": "string",
+                          "minLength": 64,
+                          "maxLength": 64,
+                          "pattern": "^[a-f0-9]{64}$"
+                        }
+                      ]
+                    },
+                    "name": {
+                      "type": "string",
+                      "minLength": 1
+                    },
+                    "paths": {
+                      "type": "array",
+                      "items": [
+                        {
+                          "type": "string",
+                          "minLength": 1
+                        }
+                      ],
+                      "minLength": 1
+                    },
+                    "terminating": {
+                      "type": "boolean"
+                    },
+                    "threshold": {
+                      "type": "integer",
+                      "minimum": 1
+                    }
+                  },
+                  "required": [
+                    "keyids",
+                    "name",
+                    "paths",
+                    "terminating",
+                    "threshold"
+                  ]
+                }
+              ]
+            }
+          },
+          "required": [
+            "keys",
+            "roles"
+          ]
+        },
+        "expires": {
+          "type": "string",
+          "format": "date-time"
+        },
+        "spec_version": {
+          "type": "string",
+          "pattern": "^[0-9]+\\.[0-9]+\\.[0-9]$"
+        },
+        "targets": {
+          "type": "object",
+          "additionalProperties": false,
+          "patternProperties": {
+            "^in-toto-metadata/[0-9a-z.]+.layout$": {
```

Comment:
Generally, neither specific targets nor delegations are expected in `targets.json`.

Reply:
Adjusted in 975c34a.